node_package/src/RSCServerRoot.ts

```
@@ -0,0 +1,111 @@
+if (!('use' in React && typeof React.use === 'function')) {
+  throw new Error('React.use is not defined. Please ensure you are using React 18 with experimental features enabled or React 19+ to use server components.');
+}
+
+const { use } = React;
+
+const createFromReactOnRailsNodeStream = (stream: NodeJS.ReadableStream, ssrManifest: Record<string, unknown>) => {
+  const transformedStream = transformRSCStream(stream);
+  return createFromNodeStream(transformedStream, ssrManifest);
+}
+
+const createSSRManifest = async (reactServerManifestFileName: string, reactClientManifestFileName: string) => {
```

ESLint

[error] Replace `reactServerManifestFileName:·string,·reactClientManifestFileName:·string` with `⏎··reactServerManifestFileName:·string,⏎··reactClientManifestFileName:·string,⏎`

(prettier/prettier)